x-pack/plugins/apm/public/components/shared/service_icons/icon_popover.tsx

```diff
@@ -22,12 +25,21 @@ interface IconPopoverProps {
   onClose: () => void;
   detailsFetchStatus: FETCH_STATUS;
   isOpen: boolean;
-  icon: {
-    type?: string;
-    size?: 's' | 'm' | 'l';
-    color?: string;
-  };
+  icon: PopoverItem['icon'];
 }
+
+const StyledPanel = styled(EuiPanel)`
+  &.serviceIcon {
+    box-shadow: ${({ theme }) => {
+      const shadowColor = theme.eui.euiShadowColor;
+      return `0px 0.7px 1.4px ${rgba(shadowColor, 0.07)},
+      0px 1.9px 4px ${rgba(shadowColor, 0.05)},
+      0px 4.5px 10px ${rgba(shadowColor, 0.05)} !important;`;
+    }};
+    height: 56px;
```

Comment:
where is this height (and the padding below) coming from? any way to make it less magic?

Reply:
I improved it a bit, the Panel component adds padding around the inner element, but it makes it a rectangle, not a square, so I set `paddingSize="none"` and only added the padding on the sides, and removed the `height`, as it is no longer needed.